### PR TITLE
New package: xmlstarlet-1.6.1

### PR DESCRIPTION
**Pull request by allan**

### CODE REVIEW

File: srcpkgs/xmlstarlet/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+makedepends="libxslt-devel"
+short_desc="Command line utilities to transform, query, validate, and edit XML"
+maintainer="allan <mail@may.mooo.com>"
+license="MIT"
+homepage="http://xmlstar.sourceforge.net/"
+distfiles="$SOURCEFORGE_SITE/xmlstar/${pkgname}/${version}/${pkgname}-${version}.tar.gz"
+checksum=15d838c4f3375332fd95554619179b69e4ec91418a3a5296e7c631b7ed19e7ca
+
+[ -n $CROSS_BUILD ] &&
+	configure_args="--with-libxml-prefix=$XBPS_CROSS_BASE/usr"
```
Comment (beefcurtains) [inline srcpkgs/xmlstarlet/template]: no xslt?

File: srcpkgs/xmlstarlet/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+depends="libxslt"
+makedepends="libxslt-devel"
+short_desc="Command line utilities to transform, query, validate, and edit XML"
+maintainer="allan <mail@may.mooo.com>"
+license="MIT"
+homepage="http://xmlstar.sourceforge.net/"
+distfiles="$SOURCEFORGE_SITE/xmlstar/${pkgname}/${version}/${pkgname}-${version}.tar.gz"
+checksum=15d838c4f3375332fd95554619179b69e4ec91418a3a5296e7c631b7ed19e7ca
+
+[ -n $CROSS_BUILD ] &&
```
Comment (xtraeme) [inline srcpkgs/xmlstarlet/template]: Remove this conditional

File: srcpkgs/xmlstarlet/template [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'xmlstarlet'
+pkgname=xmlstarlet
+version=1.6.1
+revision=1
+build_style=gnu-configure
+configure_args="--with-libxml-prefix=/usr --with-libxslt-prefix=/usr"
```
Comment (xtraeme) [inline srcpkgs/xmlstarlet/template]: `/usr` -> `$XBPS_CROSS_BASE/usr`

File: srcpkgs/xmlstarlet/template [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'xmlstarlet'
+pkgname=xmlstarlet
+version=1.6.1
+revision=1
+build_style=gnu-configure
+configure_args="--with-libxml-prefix=$XBPS_CROSS_BASE/usr
+		--with-libxslt-prefix=$XBPS_CROSS_BASE/usr"
+depends="libxslt"
```
Comment (beefcurtains) [inline srcpkgs/xmlstarlet/template]: Should be automatic, you can remove this.

**Comment by beefcurtains** [comment]
:+1:

**Pull request merged**